test/testdefs.jl

```diff
@@ -5,17 +5,8 @@ using Base.Test
 function runtests(name)
     @printf("     \033[1m*\033[0m \033[31m%-21s\033[0m", name)
     tt = @elapsed include("$name.jl")
-    @printf(" in %6.2f seconds\n", tt)
-    nothing
-end
+    @printf(" in %6.2f seconds, rss %7.2f MB\n", tt, Sys.get_rusage().ru_maxrss / 2^20)
```

Comment:
rusage is in kb so this is off by a factor of 1024 http://linux.die.net/man/2/getrusage